### PR TITLE
Add resource pkg

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
This was modeled similar to the fanuc_resource package

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
We should probably make sure spelling is consistent within this pkg: colour (filename) vs color (xacro property).

I vote _colour_ :).

File: abb_resources/urdf/common_colours.xacro [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <!-- approximations -->
+  <xacro:property name="color_abb_orange"   value="1 0.43 0 1" />
```
Comment (gavanderhoorn) [inline abb_resources/urdf/common_colours.xacro]: indent

File: abb_resources/package.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<package>
+  <name>abb_resources</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      Shared configuration data for ABB manipulators.
+    </p>
+    <p>
+      This package contains common URDF / XACRO resources used by
+      ABB related packages.
```
Comment (gavanderhoorn) [inline abb_resources/package.xml:10]: 'style rules' (if we have those) say to keep _urdf_ and _xacro_ lower case (I know the `fanuc_resources` has this upper-cased, that needs to be fixed).

**Comment by Levi-Armstrong** [comment]
I agree on the use of colour. It appears that the US is the only country that uses the English spelling color, everyone else use colour.

**Comment by Levi-Armstrong** [comment]
@gavanderhoorn, can you elaborate on your **_indent_** comment?

**Comment by gavanderhoorn** [comment]
> I agree on the use of colour. It appears that the US is the only country that uses the English spelling color, everyone else use colour.

I hope you saw my smiley :).

Anyway: perhaps _color_ would be better: the [urdf element](http://wiki.ros.org/urdf/XML/link#line-83) name is spelled without the additional 'u', and ROS uses American English predominantly when naming things.

File: abb_resources/urdf/common_colours.xacro (hunk 1 above)
Comment (gavanderhoorn) [inline abb_resources/urdf/common_colours.xacro]: `value` is misaligned with the other two lines here (by a single space).

**Comment by gavanderhoorn** [comment]
Btw: the actual _RAL codes_ for the colours ABB uses for its robots can be found on the internet (if not from ABB itself). Apparently, 'ABB Orange' is [RAL 2003](http://rgb.to/ral/2003), fi.

**Comment by Levi-Armstrong** [comment]
Yea, the smiley face is what initiated the search for the meaning of color and colour :).

@shaun-edwards, do have a preference on the spelling of color (American English) or colour (All other English speaking countries)?

**Comment by shaun-edwards** [comment]
I like the international flavor (flavour) of ROS-I

On Monday, March 2, 2015, Levi Armstrong notifications@github.com wrote:

> Yea, the smiley face is what initiated the search for the meaning of color
> and colour :).
> 
> @shaun-edwards https://github.com/shaun-edwards, do have a preference
> on the spelling of color (American English) or colour (All other English
> speaking countries)?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ros-industrial/abb_experimental/pull/20#issuecomment-76853474
> .

## 

Sent from Gmail Mobile

**Comment by Levi-Armstrong** [comment]
@shaun-edwards & @gavanderhoorn, How does this look?

File: abb_resources/urdf/common_colours.xacro [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  <!-- RAL 7032 pebble gray, controller -->
+  <xacro:property name="colour_abb_pebble_gray" value="0.7058823 0.6901961 0.6313725 1" />
+
+  <!-- RAL 7012 basalt gray (abb dark gray) , robot with dark link color -->
+  <xacro:property name="colour_abb_dark_gray" value="0.3450980 0.3647059 0.3686275 1" />
+
+  <!-- Other robot colours availble, but not limited too. -->
+  <!-- RAL 3001 signal red -->
+  <xacro:property name="colour_abb_signal_red" value="0.5960784	0.1372549 0.1372549 1" />
```
Comment (gavanderhoorn) [inline abb_resources/urdf/common_colours.xacro]: whitespace?

**Comment by gavanderhoorn** [comment]
@Levi-Armstrong: this looks great. Very nice. Did you contact ABB to get info on the colours?

+1 to merge, imo.

**Comment by gavanderhoorn** [comment]
Once we get this merged, 'all' released support pkgs should be updated to make use of this.

**Comment by gavanderhoorn** [comment]
O, please merge fixups into the first commit.

**Comment by Levi-Armstrong** [comment]
@gavanderhoorn: Yea, I contacted ABB and they sent me the colour information. I also found a ABB technote that included the other available colours, so I assume these are the most common colours ordered other than the standard so I included them.

File: abb_resources/urdf/common_colours.xacro (hunk 3 above)
Comment (Levi-Armstrong) [inline abb_resources/urdf/common_colours.xacro]: Not sure why it is showing up that way, but if you view the file it is not present.

Comment (gavanderhoorn) [inline abb_resources/urdf/common_colours.xacro]: tabs vs spaces. That line actually has a tab in it.

**Pull request merged**